### ISSUE TITLE
gecko_ia2: NVDA does not announce text selection changes in edit fields

### ISSUE THREAD

**Issue by nvdaTrac** [nvaccess/test_nvdaTracImport#187]
**Reported by Iris on 12 Jun 2008 04:19 UTC**
When one is typing in the edit field on Google, for instance, NVDA will not announce the current state of the text when one needs to select all that you've typed or just a word that you've mistyped.

**Comment by nvdaTrac** [comment]
**Comment by jteh on 12 Jun 2008 04:52 UTC**
Note that this does not occur if you focus the field while virtual buffer pass-through mode is on. I'm working on a fix now.

**Comment by nvdaTrac** [comment]
**Modified by jteh on 12 Jun 2008 04:53 UTC**

**Comment by nvdaTrac** [comment]
**Comment by jteh on 12 Jun 2008 05:15 UTC**
Fixed in r2118.
